services/workflows-service/.env.example

```diff
@@ -35,3 +35,6 @@ WEB_UI_SDK_URL=http://localhost:5202
 #HASHING_KEY_SECRET="$2b$10$FovZTB91/QQ4Yu28nvL8e."
 HASHING_KEY_SECRET_BASE64=JDJiJDEwJDNFeWtwWEs4QkdiczlRaWFwLkM4Vk8=
 NOTION_API_KEY=secret
+REDIS_HOST=localhost
```

Comment:
I think we would want something like USE_REDIS - to change between using redis to im-mem implmenation